store/sqlstore/supplier.go

```diff
@@ -164,7 +164,12 @@ func NewSqlSupplier(settings model.SqlSettings, metrics einterfaces.MetricsInter
 		os.Exit(EXIT_CREATE_TABLE)
 	}
 
-	UpgradeDatabase(supplier)
+	err = UpgradeDatabase(supplier, model.CurrentVersion)
+	if err != nil {
+		mlog.Critical("Failed to upgrade database", mlog.Err(err))
+		time.Sleep(time.Second)
+		os.Exit(EXIT_GENERIC_FAILURE)
```

Comment:
We used to have a habit of exiting with distinct error codes, but this hasn't proven useful. A while back, I introduced `EXIT_GENERIC_FAILURE` to take the place of "new" codes as such.